packages/mui-material/src/StepIcon/StepIcon.spex.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+import StepIcon from '@mui/material/StepIcon';
+
+<StepIcon icon={<div>icon</div>} />;
+<StepIcon icon={<div>icon</div>} titleAccess="title" />;
```

Comment:
test for the fix

Reply:
The extension of the file should be `spec.tsx` (currently `spex.tsx`)

Reply:
updated here https://github.com/mui/material-ui/pull/44337/commits/7fffa51a9a1f851f4ea42322f2067f84abf463f3